erigon-lib/commitment/hex_patricia_hashed.go

```diff
@@ -121,6 +121,21 @@ func (cell *cell) reset() {
 	cell.hashedExtLen = 0
 	cell.extLen = 0
 	cell.hashLen = 0
+	for i := range cell.hashedExtension {
```

Comment:
can do `clear(cell.hashedExtension)`

Reply:
Cannot use clear on  arrays, only on slices and maps :

```
invalid argument: cannot clear cell.hashedExtension (variable of type [128]byte): argument must be (or constrained by) map or slice
```

Reply:
`clear(cell.hashedExtension[:])`